docs/content/docs/user-docs/install.md

```diff
@@ -59,19 +59,28 @@ helm pull oci://$CHART_REPO --version $RELEASE_VERSION -d $CHART_EXPORT_PATH
 tar xvf $CHART_EXPORT_PATH/$CHART_PACKAGE -C $CHART_EXPORT_PATH
 ```
 
+{{% hint type="info" title="s3-controller version" %}}
+Above commands will download the latest version of `s3-controller`. To select a
+different version, change `RELEASE_VERSION` variable and execute the commands again.
+{{% /hint %}}
+
 Once the Helm chart is downloaded and exported, you can install a particular ACK service controller using the `helm install` command:
 
 ```bash
 export ACK_K8S_NAMESPACE=ack-system
+export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
+export AWS_REGION=us-west-2
 
 helm install --create-namespace --namespace $ACK_K8S_NAMESPACE ack-$SERVICE-controller \
+    --set aws.account_id="$AWS_ACCOUNT_ID" \
+    --set aws.region="$AWS_REGION" \
     $CHART_EXPORT_PATH/$SERVICE-chart
 ```
 
 The `helm install` command should return relevant installation information:
 
 ```bash
-helm install --namespace $ACK_K8S_NAMESPACE ack-$SERVICE-controller $CHART_EXPORT_PATH/$SERVICE-chart
+helm install --namespace $ACK_K8S_NAMESPACE ack-$SERVICE-controller --set aws.account_id="$AWS_ACCOUNT_ID" --set aws.region="$AWS_REGION" $CHART_EXPORT_PATH/$SERVICE-chart
```

Comment:
How about we split this long line across multiple lines like above on lines 74-78

Reply:
I hope that we can do away with manually setting the account ID, seeing as Vijay has the PRs out for defaulting it. I'd prefer to wait for that and leave it out.

Reply:
To clarify why I'm against this in the documentation:
This method currently assumes that you have authenticated with the same account as the one you are using for ACK. It is possible that you have been giving access to an EKS cluster in another account, or don't have programmatic (CLI) access to the same account in which the cluster resides. In that case, `get-caller-identity` will either return the wrong value or no value. I personally don't want to debug that nightmare of a scenario.

Reply:
Interesting perspective on this @RedbackThomson , But solely relying on sts `GetCallerIdentity` to find AWS AccountId(which is right thing IMO) will require moving `irsa.md` and `authorization.md` doc before `install.md` in order. 
This is because if IAM is not configured for controller-pod, ack runtime will not be able to find AWS AccountId.

Is that something we are okay with?

Reply:
If the runtime can't find an account ID, will the pod crash loop? If so, then it's fine to have authentication installation afterward - it will crash loop until IRSA (or otherwise) is configured properly. In fact this might be a good indication that the user has configured their authentication properly.

Reply:
Good point. But I will leave this as is right now because we have are seeing some [customer friction](https://github.com/aws-controllers-k8s/community/issues/990)

Once my changes are merged for runtime and code-gen and new controller is released for s3, I will come back and remove accountid option from here.